src/Components/GameScreenComponents/Column.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import "../../styles/GameScreen.css"
+
+interface myProps{
```

Comment:
By convention, type names start with capital letters. It's also helpful to give the type a more descriptive name such as `ColumnProps`